staff/luis-garrido/gamma-template/style.sass

```diff
@@ -0,0 +1,276 @@
+@import url('https://fonts.googleapis.com/css?family=Sarabun:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i')
+
+$gamma-green: rgb(28,170,152)
+$border-gray: solid 1px rgb(204,204,204)
+
+html, 
+body
+    margin: 0px
```

Comment:
When you set margin or padding to 0, you don't need to put px. you can just write margin: 0.

Reply:
Done! Thanks for the tip!